airflow/providers/openlineage/provider.yaml

```diff
@@ -77,6 +77,14 @@ config:
         example: "airflow.operators.bash.BashOperator;airflow.operators.python.PythonOperator"
         default: ""
         version_added: 1.1.0
+      opt_in:
```

Comment:
`force_per_dag_opt_in`?

Reply:
1. `explicit_lineage_control`
2. `selective_lineage_control`
3. `lineage_on_demand`
 ?

Reply:
Both `explicit_lineage_control` and  `selective_lineage_control` work for me. `force_per_dag_opt_in` limits scope per DAG (which isn't true, you can control per task too). `lineage_on_demand` sounds misleading - on what demand, e.g. is this lineage whenever you choose to emit it?

Reply:
I think the name should indicate what setting it to `true` or `false` will do. `selective control` and `explicit control` mean nothing tbh

Reply:
What about `disable_unless_decorated`?

Reply:
More precise:`disabled_unless_with_context` or `enable_with_context_only`?

Reply:
What is the `context` then?

Reply:
Python context manager (`with`). This may be more explicit:
`disabled_unless_with_context_manager`

Reply:
Along the lines of what @kacpermuda proposed, what about... `selective_enable`? it's already in the `openlineage` configuration section, so we probably do not need to add lineage to its name

Reply:
I changed it to `selective_enable`. Thanks @tatiana for the suggestions!